### DISCUSSION TITLE
It would be useful to make CTkOptionMenu `dropdown_arrow` optional

### DISCUSSION THREAD

**Discussion by LucianoSaldivia** [TomSchimansky/CustomTkinter#1382] [Ideas]
I'm trying to make a title bar like VSCode has (image below), so i deleted the default title bar on my app and created my own.
The thing is, when trying to create option menues, the arrow just looks odd to have there.

![image](https://user-images.githubusercontent.com/32425456/227754659-78c787c0-a83f-4561-8690-819aa7297607.png)

Adding a bool to the __init__() method would be nice, so you could choose whether to draw it or not, and maybe set `True` as the default value. Reading the `ctk_optionmenu.py` file, i think this would just add a simple `if`-statement, but i'm not entirely sure about it.

**Comment by avalon60** [comment]
That looks more like a Tkinter menu. You can interrogate the CustomTkinter theme to grab the colours to use on a regular tkinter.Menu. I do this with one of my apps.
https://pythonbasics.org/tkinter-menu/#:~:text=The%20tkinter%20menu%20is%20a,a%20Python%20method%20is%20called.

**Reply by LucianoSaldivia** [reply]
Thanks for answering, i knew about Tkinter's `Menu` but what do you mean when you say "interrogate the CustomTkinter theme to grab the colours" ? Could you show me an example?
Also, would that automatically update upon the `theme_changed` event?
And last, wouldn't that just grab the colours and nothing else in the style? 

I can see some differences, for example you can see in the image i provided, that the `File` menu is highlighted and you can see it has **rounded corners**, is this possible with a Tkinter `Menu` ?

**Comment by LucianoSaldivia** [comment]
I just realized that an OptionMenu without an arrow is not what i need, since what's written inside of it as the "selected option", must always be one of the options, and i need it to be some sort of 'title' or 'name' of the menu.
So what i'm actually looking for, is to be able to add a dropdown menu to a button or a label. I'll go into more details after reading the `dropdown_menu.py` inside "customtkinter/windows/widgets/core_widgets_classes/".
